proxy/src/main/java/com/wavefront/agent/handlers/DeltaCounterAccumulationHandlerImpl.java

```diff
@@ -0,0 +1,168 @@
+package com.wavefront.agent.handlers;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.RemovalListener;
+import com.google.common.util.concurrent.AtomicDouble;
+import com.wavefront.api.agent.ValidationConfiguration;
+import com.wavefront.common.Clock;
+import com.wavefront.common.HostMetricTagsPair;
+import com.wavefront.data.ReportableEntityType;
+import com.wavefront.data.Validation;
+import com.wavefront.ingester.ReportPointSerializer;
+import com.yammer.metrics.Metrics;
+import com.yammer.metrics.core.DeltaCounter;
+import com.yammer.metrics.core.Histogram;
+import com.yammer.metrics.core.MetricName;
+import com.yammer.metrics.core.MetricsRegistry;
+import org.apache.commons.lang.math.NumberUtils;
+import java.util.Collection;
+import java.util.Random;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import com.yammer.metrics.core.Counter;
+import javax.annotation.Nullable;
+import wavefront.report.ReportPoint;
+import static com.wavefront.data.Validation.validatePoint;
+import static com.wavefront.sdk.common.Utils.metricToLineData;
+import com.yammer.metrics.core.Gauge;
+
+/**
+ * Handler that processes incoming DeltaCounter objects, validates them and hands them over to one of
+ * the {@link SenderTask} threads.
```

Comment:
Nit: update the description so it reflects what we're actually doing here - since we're not passing the data through